news/2018-01-31-uioPlus.html

```diff
@@ -0,0 +1,168 @@
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>Introducing UI Options Plus (UIO+)</h3>
+                <time class="floe-date" datetime="2018-01-31">January 31, 2018</time>
+                <p>
+                    User Interface Options also known as Learner Options provides individuals with the ability to customize a web
+                    site to meet their personal needs and preferences. For example, if the contrast of this page isn't quite right, you can use the tab at the top
+                    to modify how things are displayed. But what do you do when you are on a site that doesn't incorporate UI Options or
+                    other personalization methods?
+                </p>
+
+                <p>
+                    We have recently published a beta of <a href="https://chrome.google.com/webstore/detail/ui-options-plus-uio+/okenndailhmikjjfcnmolpaefecbpaek">User Interface Options Plus (UIO+)</a> to the Chrome Web Store.
+                    UIO+ is a new tool to enable the personalization experience of UI Options as a browser extension.
+                    Preferences can be set via the UIO+ adjuster panel or, if on a <a href="https://gpii.net/">GPII (Global Public Inclusive Infrastructure)</a> enabled machine, from a keyed in preference set.
+                </p>
+
+                <p>
+                    The following preferences are currently supported:
+
+                    <ul>
+                        <li>Text Size</li>
+                        <li>Line Space</li>
+                        <li>Contrast</li>
+                        <li>Right-Click to Select</li>
+                        <li>Selection Highlight</li>
+                        <li>Text-to-Speech (requires the <a href="https://chrome.google.com/webstore/detail/click2speech/djfpbemmcokhlllnafdmomgecdlicfhj">click2speech</a> extension)</li>
+                        <li>Reading Mode</li>
+                        <li>Table of Contents</li>
+                        <li>Enhance Inputs</li>
+                        <li>Dictionary (requires the <a href="https://chrome.google.com/webstore/detail/google-dictionary-by-goog/mgijmajocgfcbeboacabfgobmjgjcoja">Google dictionary</a> extension)</li>
+                    </ul>
+
+                    <em><strong>Note</strong>: The ability to apply an adaptation will vary from page to page</em>
+                </p>
+
+                <p>
+                    UIO+ is still early in development. We aim to improve/expand preference options and, in the future, support additional browsers.
```

Comment:
Rephrase: "UIO+ is still early in development. We aim to improve/expand preference options and, in the future, support additional browsers."

To: "UIO+ is still early in development. We aim to improve/expand preference options and support additional browsers in the future.